src/select-menu/src/SelectMenu.js

```diff
@@ -36,65 +36,62 @@ const getEmptyView = (close, emptyView) => {
   return {}
 }
 
-const SelectMenu = memo(
-  forwardRef((props, ref) => {
```

Comment:
Not clear what the ref should be attached to, and Popover doesnt forward refs anyway.